src/main/resources/listLookup.json

```diff
@@ -449,8 +449,8 @@
         "isWeeklyList": true
     },
     "FTT_LR_WEEKLY_HEARING_LIST": {
-        "friendlyName": "First-tier Tribunal (Lands Registration Tribunal) Weekly Hearing List",
-        "welshFriendlyName": "Rhestr o Wrandawiadau Wythnosol Tribiwnlys Haen Gyntaf (Tribiwnlys Cofrestru Tir)",
+        "friendlyName": "First-tier Tribunal (Land Registration Tribunal) Weekly Hearing List",
+        "welshFriendlyName": "Tribiwnlys Haen Gyntaf (Tribiwnlys Cofrestru Tir)",
```

Comment:
The welsh name reads "First-tier Tribunal (Land Registration Tribunal)".  The words "weekly hearing list" is missing